src/utils.js

```diff
@@ -17,4 +17,6 @@ export const validateJSONPath = path => {
   if (!isJSONPath(path)) {
     throw new Error(`${INVALID_JSON_PATH_ERROR}: "${path === '' ? '' : path}"`)
   }
+
+  return true
```

Comment:
Related with the comment above, and in any part of the code is used.

Reply:
Currently the true value returned has no use in the code base (currently), was an instinctual change so as not to be a void function. Can be reverted.